### ISSUE TITLE
Does this plugin support Team Google Drives?

### ISSUE THREAD

**Issue by victsant** [xuopled/gatsby-source-google-docs#20]
I am trying to integrate a team google drive instead of the default user google drive and it's not reading the files for the team drive.

**Comment by xuopled** [comment]
I don't know how it works but maybe you can change the default scope:

```
scope: [
    "https://www.googleapis.com/auth/documents.readonly",
    "https://www.googleapis.com/auth/drive.metadata.readonly",
  ],
```

**Comment by victsant** [comment]
@xuopled,

I just enabled team drives by editing your repo by adding these two lines in bold to drive.files.list call.

`drive.files.list({
        **includeTeamDriveItems: true,
        supportsAllDrives: true,**
        q: `'${folderId}' in parents and (mimeType='${MIME_TYPE_FOLDER}' or mimeType='${MIME_TYPE_DOCUMENT}') and trashed = false`,
        fields: `files(id, mimeType, name, description${fields ? `, ${fields.join(", ")}` : ""})`
      }`

Do you think that is something you can add to this or should I just fork your repo?

**Comment by xuopled** [comment]
Thank you for your PR. I just merged it and published v1.8.0
